### PR TITLE
Rename Labels on Work Tabs and File Set Edit

### PR DESCRIPTION
**Pull request by mtribone**
Adds content for tabs to sufia.en.yml and updates text. Changes views to use translation. Remove icons from tabs. Update tests to reflect name changes.

Changes based on user testing and accessibility testing with Christian and Michele.

### CODE REVIEW

**Comment by mtribone** [comment]
screen shot 2017-03-17 at 3 01 00 pm

**Comment by mtribone** [comment]
screen shot 2017-03-17 at 3 01 19 pm

**Comment by mtribone** [comment]
screen shot 2017-03-17 at 3 01 55 pm

File: app/views/curation_concerns/base/_guts4form.html.erb [hunk 1]
```diff
@@ -0,0 +1,47 @@
+<% # we will yield to content_for for each tab, e.g. :files_tab %>
```
Comment (carolyncole) [inline app/views/curation_concerns/base/_guts4form.html.erb:1]: @mtribone Did we need this entire override?  Was this to just change the tab name?

Comment (mtribone) [reply app/views/curation_concerns/base/_guts4form.html.erb:1]: It was to remove the icons. I may be able to use some CSS trickery to hide them, but I was able to remove them everyplace else except for the main form.

**Pull request merged**